lib/smack/SmackInstGenerator.cpp

```diff
@@ -594,6 +594,12 @@ void SmackInstGenerator::visitCallInst(llvm::CallInst& ci) {
     WARN("ignoring llvm.debug call.");
     emit(Stmt::skip());
 
+  } else if (name.find("llvm.expect.") == 0) {
```

Comment:
The style of this if condition should match the others in this if-else-if block. Please update this.

Reply:
Do you mean compare it to string::npos? My reasoning is that we want to be sure "llvm." is at the beginning of the function name. This guarantees that we are actually replacing an LLVM function, and not some other function. Perhaps I should change llvm.dbg to compare to 0, but it seems unlikely to matter either way.

Reply:
I see, good point. I think what you wrote could probably be said for all of these conditions, including the ones below. So to be consistent, let's still use `!= std::string::npos` here.

Reply:
Done.